.github/workflows/julia-concept-ci.yml

```diff
@@ -0,0 +1,31 @@
+name: Julia Concept CI
+
+on:
+  push:
+    paths:
+    - 'languages/julia/config.json'
+    - 'languages/julia/reference/concepts.csv'
+    - 'languages/julia/reference/exercise-concepts/**'
+    - 'languages/julia/exercises/concept/**'
+  pull_request:
+    paths:
+    - 'languages/julia/config.json'
+    - 'languages/julia/reference/concepts.csv'
+    - 'languages/julia/reference/exercise-concepts/**'
+    - 'languages/julia/exercises/concept/**'
+
+jobs:
+  concept-test:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: "Set up Julia"
+        uses: julia-actions/setup-julia@v1
```

Comment:
I remember that we wanted to refer to git hashes instead of versions for security reasons. Was that correct?

Reply:
Yes and no. This particular workflow doesn't use any secrets and doesn't have write access to the repo, so it's not necessary to use git hashes. The hashes are only necessary when secrets are involved to protect against someone getting access to secrets by changing the action.

Reply:
Okay cool!